src/MovieList.jsx

```diff
@@ -53,7 +53,7 @@ function MovieList({
       .catch((error) => console.error("Error fetching data:", error));
   }, [page, fetchURL, sortSelection, genreSelection]);
   //load API data into movieCard containers
-  const divs = data?.results?.map((movie, index) => (
+  const divs = data?.results?.slice(0, 6).map((movie, index) => (
```

Comment:
Is 6 the page size? I think it should be in a constant, preferably in an external file so it can be imported and used in other places

Reply:
Probably movieCards is a better name for this var than divs.

Reply:
Sounds good!